src/components/EngineManageDialog.vue

```diff
@@ -150,31 +155,44 @@
           >
             <div class="q-pl-md q-mt-md flex">
               <img
+                v-if="engineIcons[selectedId]"
                 :src="engineIcons[selectedId]"
                 :alt="engineInfos[selectedId].name"
                 class="engine-icon"
               />
+              <div class="title-engine-icon-container inline-block" v-else>
+                <q-avatar rounded color="primary" size="2rem">
+                  <span class="text-display-on-primary"> ? </span>
+                </q-avatar>
+              </div>
               <div class="text-h5 q-ma-sm">
                 {{ engineInfos[selectedId].name }}
               </div>
             </div>
 
             <div class="no-wrap q-pl-md">
               <ul>
-                <li>バージョン：{{ engineVersions[selectedId] }}</li>
                 <li>
-                  URL：<a
+                  バージョン：{{
+                    engineVersions[selectedId] ?? "（取得に失敗しました）"
```

Comment:
（すみません。。`template`内で`??`使うとシンタックスハイライトが壊れてコードが読みにくくなってしまうので、こんな感じに修正お願いしても良いでしょうか。。。 🙇  ）
（ご面倒でしたらあとでこちらでよしなに変更します！）
```suggestion
                    engineVersions[selectedId] ? engineVersions[selectedId] : "（取得に失敗しました）"
```